doc/5_saving.md

```diff
@@ -77,5 +77,6 @@ The following properties set in `SparkConf` can be used to fine-tune the saving
      will adjust the number of rows based on the amount of data in each row  
   - `spark.cassandra.output.batch.size.bytes`: maximum total size of the batch in bytes; defaults to 16 kB.
   - `spark.cassandra.output.concurrent.writes`: maximum number of batches executed in parallel by a single Spark task; defaults to 5
+  - `spark.cassandra.output.consistency.level`: consistency level for writing; defaults to ONE.
```

Comment:
You're right, also there is an open issue regarding this, I created a new pull request here: https://github.com/datastax/spark-cassandra-connector/pull/533
We can close this one then